RELEASE/scripts/autoscend/quests/level_09.ash

```diff
@@ -210,6 +210,19 @@ void prepareForSmutOrcs()
 	
 	// This adds a tonne of damage and NC progress
 	buffMaintain($effect[Triple-Sized]);
+
+	// pull a keepsake box if we can
+	pullXWhenHaveY($item[smut orc keepsake box], 1, 0);
```

Comment:
This would be better in `L9_chasmBuild()`. It already has code to use keepsake boxes. Add the pull attempt right before that